prediction/prediction_output.py

```diff
@@ -20,3 +20,25 @@ def get_outcome_prob(self, outcome):
 
     def get_nonzero_prob_outcomes_list(self):
         return list(self.score_dict.items())
+
+
+class TournamentOutput(object):
```

Comment:
put these two classes in a separate file named tournament_output.py
and think about other useful public interfaces to expose.

e.g. get_tournamnet_structure or something?